src/qt/forms/intro.ui

```diff
@@ -219,6 +219,13 @@
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QLabel" name="lblPruneSuffix">
+       <property name="buddy">
+        <cstring>pruneGB</cstring>
+       </property>
```

Comment:
7d40a4805229618a364dc53a84cd5af2ac273b7d
Where is this used?

Reply:
It's here for correctness. I don't know if there's a platform that acts on it.

Reply:
> It's here for correctness. I don't know if there's a platform that acts on it.

I cannot parse this message. Correctness of what?

Reply:
@luke-jr 

From [`QLabel::setBuddy`](https://doc.qt.io/qt-5/qlabel.html#setBuddy):
> The buddy mechanism is only available for `QLabel`s that contain text in which one character is prefixed with an ampersand, '&'. This character is set as the shortcut key.

You do not set a shortcut key, therefore linking `lblPruneSuffix` to `pruneGB` buddy is noop, and should be dropped.